docs/REST API/rest-api.md

```diff
@@ -5646,6 +5688,36 @@ Get aggregate status counts of profiles for to macOS and Windows hosts that are
 }
 ```
 
+### Get OS setting (configuration profile) status
+
+Get status counts of a single OS settings (configuration profile) enforced on hosts.
+
+`GET /api/v1/fleet/configuration_profile/:profile_uuid/status`
+
+#### Parameters
+
+| Name                      | Type   | In    | Description                                                               |
+| ------------------------- | ------ | ----- | ------------------------------------------------------------------------- |
+| profile_uuid                   | string | query | **Required**. The UUID of configuration profile.             |
+
+#### Example
+
+`GET /api/v1/fleet/configuration_profile/f663713f-04ee-40f0-a95a-7af428c351a9/status`
```

Comment:
@mna @marko-lisica just a heads up that we should add an `s` the `configuation_profile` to make the endpoint `configuration_profiles/:uuid/status`.